chainermn/links/batch_normalization.py

```diff
@@ -10,7 +10,9 @@
 from chainermn.functions.batch_normalization import \
     get_communication_backend
 from chainermn.functions.batch_normalization import \
-    MultiNodeBatchNormalizationFunction
+    multinode_bn_backend_selector
+from chainer.functions.normalization.batch_normalization \
+    import BatchNormalization
```

Comment:
Lets' avoid importing objects, as per [the OpenStack Style Guideline](https://docs.openstack.org/hacking/latest/user/hacking.html#imports).

Reply:
Addressed in 4c2fe6c